junit5-testing/pom.xml

```diff
@@ -122,6 +122,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+		<dependency>
+  <groupId>com.google.code.gson</groupId>
+  <artifactId>gson</artifactId>
+  <version>2.8.5</version> <!-- check latest version for GSON -->
+</dependency>
```

Comment:
Hi, just trying to figure out - why do you need it here to be served as a transitive dependency? How `gson` lib is linked with the `Jupiter` runners?